src/helpers.js

```diff
@@ -387,6 +387,14 @@ export default function Helpers(mpInstance) {
         return filteredUserAttributes;
     };
 
+    this.isFilteredUserAttribute = function(userAttributeKey, filterList) {
+        const hashedUserAttribute = self.generateHash(userAttributeKey);
+        if (filterList && !self.inArray(filterList, hashedUserAttribute)) {
+            return true;
+        }
+        return false;
```

Comment:
Since you're returning a boolean for this, you can just return the whole conditional as the result, which will be a boolean.

```suggestion
        return filterList && !self.inArray(filterList, hashedUserAttribute));
```